docs/docs/dev_docs/tutorials/token_portal/typescript_glue_code.md

```diff
@@ -47,8 +40,8 @@ Open `cross_chain_messaging.test.ts` and paste the initial description of the te
 
 ```typescript
 import { expect, jest} from '@jest/globals'
-import { AccountWallet, AztecAddress, DebugLogger, EthAddress, Fr, computeAuthWitMessageHash, createDebugLogger, createPXEClient, waitForPXE } from '@aztec/aztec.js';
-import { getInitialTestAccountsWallets } from '@aztec/accounts/testing';
+import { AccountWallet, AztecAddress, DebugLogger, EthAddress, Fr, computeAuthWitMessageHash, createDebugLogger, createPXEClient, waitForSandbox } from '@aztec/aztec.js';
+import { getSandboxAccountsWallets } from '@aztec/accounts/testing';
```

Comment:
Can we update this code block to import code directly from [the tests](https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/end-to-end/src/e2e_cross_chain_messaging.test.ts)?

Reply:
the tests themselves have a MUCH more complicated setup than the e2e code we use in the tutorials. it requires 4 extra files with a bunch of new imports. the tutorial setup code is not referenced anywhere else in the codebase

the only solutions are to overcomplicate the tutorial or manually keep an eye on ~15 lines of code. i'm happy to test every release - it's all in devrel repo anyway

Reply:
ok cool, works for me!